### PR TITLE
Update SECURITY.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Minor grammar edits

**Special notes for your reviewer**: This is another topic that should really be put somewhere more prominent. In particular, it would be good in the Grafana documentation so that it comes up in a search. Perhaps not the email and key, but at least instructions regarding which file in the repo to go to.

It also would not be out of place in README.md.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
-Reporting Security Issues
+# Reporting security issues
 ------------------
```
Comment (marcusolsson) [inline SECURITY.md]: When using the hash tag, we can remove these hypens.

**Review by marcusolsson — CHANGES_REQUESTED** [review]

**Comment by marefr** [comment]
It is in the root to show up here: https://github.com/grafana/grafana/security/policy

**Comment by oddlittlebird** [comment]
Ah, I see. I suggest at some point we add it to the official documentation. Or at least add a link to the security policy. "To report a security vulnerability, see the security policy on GitHub (link)."

**Review by marcusolsson — APPROVED** [review]

**Comment by marcusolsson** [comment]
Thanks! 👍

**Pull request merged**